test/core/data.wast

```diff
@@ -382,6 +382,14 @@
   "type mismatch"
 )
 
+(assert_invalid
```

Comment:
Is this the same test as in `elem.wast`? Should this instead be testing a data segment?

Reply:
Ah yes, indeed. Fixed.